libr/bin/obj.c

```diff
@@ -55,8 +55,9 @@ R_API RBinObject *r_bin_object_new(RBinFile *binfile, RBinPlugin *plugin, ut64 b
 		if (sz < bsz) {
 			bsz = sz;
 		}
-		o->bin_obj = plugin->load_bytes (binfile, bytes + offset, sz,
+		plugin->load_bytes (binfile, &o->bin_obj, bytes + offset, sz,
```

Comment:
you have to get the `bool result` from load_bytes and check it at https://github.com/radare/radare2/pull/11679/files#diff-ce3aeea66b7b1f6e50aaa345b1963acdR61, instead of checking o->bin_obj.

Reply:
Ok, I'll change that. I didn't change it before because I was avoiding to make those changes in this PR, that already changed a lot, and to keep most of the code close to the original.

Reply:
Yes but this is causing the issues you are seeing in travis

Reply:
I did change it, but travis is still red.